src/main/java/com/github/splendor_mobile_game/database/InMemoryDatabase.java

```diff
@@ -73,7 +73,7 @@ public ArrayList<Room> getAllRooms() {
 
     @Override
     public void loadCards() {
-        String csvFile = "CardDatabase.csv";
+        String csvFile = "src/main/java/com/github/splendor_mobile_game/database/CardDatabase.csv";
```

Comment:
This file should be stored in some new top level directory like `resources`. Definitively not in `src`.